CustomApps/lyrics-plus/index.js

```diff
@@ -464,14 +510,14 @@ class LyricsContainer extends react.Component {
             } else if (mode === SYNCED && this.state.synced) {
                 activeItem = react.createElement(CONFIG.visual["synced-compact"] ? SyncedLyricsPage : SyncedExpandedLyricsPage, {
                     trackUri: this.state.uri,
-                    lyrics: this.state.synced,
+                    lyrics: CONFIG.visual["translate"] && translatedLyrics ? translatedLyrics : this.state.synced,
                     provider: this.state.provider,
                     copyright: this.state.copyright,
                 });
             } else if (mode === UNSYNCED && this.state.unsynced) {
                 activeItem = react.createElement(UnsyncedLyricsPage, {
                     trackUri: this.state.uri,
-                    lyrics: this.state.unsynced,
+                    lyrics: CONFIG.visual["translate"] && translatedLyrics ? translatedLyrics : this.state.synced,
```

Comment:
shouldn't it be
```suggestion
                    lyrics: CONFIG.visual["translate"] && translatedLyrics ? translatedLyrics : this.state.unsynced,
```
since it's about unsyced lyrics, not synced ones